sync-data-to-another-database/README.md

```diff
@@ -0,0 +1,195 @@
+# Synchronize Kuzzle with another database
+
+## Introduction
+
+Kuzzle uses Elasticsearch as a database, which allows it to offer very good search performance on large volumes.  
+
+However, it can also be useful to dump Kuzzle data into more traditional databases, for example if you wanted to store and analyse historical data.  
+[Cassandra](https://cassandra.apache.org/) is a distributed NoSQL database designed to handle large volumes of data.  
+
+In this How-To, we will show you how to develop a Kuzzle Plugin that synchonizes Kuzzle's Elasticsearch database with Cassandra.  
+For this example we will use data from the NYC Yellow Taxi dataset.  
+
+## Architecture
+
+We will be using the Kuzzle stack (Kuzzle, Elasticsearch and Redis) along with an additional container that runs the Cassandra database. (Check [docker-compose.yml](docker-compose.yml) for more details)
```

Comment:
`We will be using the Kuzzle stack (Kuzzle, Elasticsearch and Redis) along with an additional container that runs the Cassandra database (check [docker-compose.yml](docker-compose.yml) for more details).`